Patch to allow the JDK API location to be passed to Ant using -Djdk.api

--- build.xml.orig	2016-06-20 15:06:43 UTC
+++ build.xml
@@ -355,6 +355,7 @@
         <include name="**"/>
         <exclude name="org/jgrapht/experimental/**"/>
       </packageset>
+      <link offline="true" packagelistLoc="${jdk.api}" href="${jdk.api}"/>
     </javadoc>
   </target>
 
